src/clipboard.js

```diff
@@ -87,14 +90,19 @@ export function filterHtmlElements (elem) {
       return ''
     }
 
+    // Keep internal relative links relative (on paste).
+    if (keepInternalRelativeLinks && child.nodeName === 'A' && child.href) {
+      const stripInternalHost = child.href.replace(window.location.origin, '')
```

Comment:
I would use `getAttribute('href')` here. That sounds more consistent. And `child.href` always adds `window.location.origin` and then you strip it away again which is not really necessary.

Reply:
I don't really get this comment, the sense of this PR is to add the option to remove/strip `window.location.origin` from the `href` attribute  give that it's an relative link